readme.md

```diff
@@ -172,19 +180,20 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:**
 
----
+It will ensure you add the data required to complete the object in your collections.
```

Comment:
The question is asking how to make use of these specific messages. You can capture the error with a `try/catch` block or a `.catch()` method. Then, you can loop over data inside of the error.